views/js/qtiCreator/widgets/interactions/graphicOrderInteraction/states/Question.js

```diff
@@ -214,15 +212,26 @@ define([
 
         $form.html(formTpl({
             baseUrl         : options.baseUrl,
-            maxChoices      : parseInt(interaction.attr('maxChoices')),
-            minChoices      : parseInt(interaction.attr('minChoices')),
-            choicesCount    : _.size(interaction.getChoices()),
             data            : interaction.object.attr('data'),
             width           : interaction.object.attr('width'),
             height          : interaction.object.attr('height'),
             type            : interaction.object.attr('type')
         }));
 
+        //controls min and max choices
+        minMaxComponentFactory($form.find('.min-max-panel'), {
+            min : { value : _.parseInt(interaction.attr('minChoices')) || 0 },
+            max : { value : _.parseInt(interaction.attr('maxChoices')) || 0 },
+            upperThreshold : _.size(interaction.getChoices())
+        }).on('render', function(){
+            var self = this;
+            widget.on('choiceCreated choiceDeleted', function(data){
+                if(data.interaction.serial === interaction.serial){
+                    self.updateThresholds(1, _.size(interaction.getChoices()));
```

Comment:
It seems that this is not working as expected: set 2 choices, set max to 2, remove 1 choice -> max is still 2